### PR TITLE
[qt] Update coin control and smartfee labels

### PR DESCRIPTION
**Pull request by maflcko**
This will sync the qt labels with the wallet's fee code.

### CODE REVIEW

**Comment by luke-jr** [comment]
So the idea here is to use the min relay fee instead of the wallet's configured fee, whichever is higher?

**Comment by MarcoFalke** [comment]
Yes, it's just what the wallet already does. This should just make qt consistent again. C.f. 
* 037b4f1485404a69f96ac6c448a97d21315c8663 (Make mining fee policy match relay fee policy) and 
* aa279d613152e87ea25edfdf76c86779c0632f18 (Enforce minRelayTxFee on wallet created tx)

**Comment by jgarzik** [comment]
ut ACK

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -2120,6 +2120,11 @@ bool CWallet::CommitTransaction(CWalletTx& wtxNew, CReserveKey& reservekey)
     return true;
 }
 
+CAmount CWallet::GetRequiredFee(unsigned int nTxBytes)
+{
+    return std::max(minTxFee.GetFee(nTxBytes), ::minRelayTxFee.GetFee(nTxBytes));
```
Comment (jonasschnelli) [inline src/wallet/wallet.cpp]: Sad to see another coupling between the Wallet and the Mempool/Node. But it's already all over.
Ideally the wallet should not directly access global mempool variables instead it should get data over a general interface.

**Comment by jonasschnelli** [comment]
Makes sense.
utACK.

**Comment by MarcoFalke** [comment]
@morcos This is addressing your issue #5202. Mind to review?

**Comment by morcos** [comment]
I think most of the pull makes sense, but I'd save removing the AbsurdFee check until we replace it with something better.  Having the only effective high fee check be the ATMP check is dicey since that depends on minRelayTxFee which people might be adjusting for other reasons.  Also I'm not sure how this is meant to work with #6726.  I don't like the idea of having less and less checks for accidental high fees. I just think we should rework them to be better organized.

**Comment by laanwj** [comment]
I agree with @morcos. The absurdfee check is just a belt-and-suspenders check, in case something goes wrong due to a bug in another part of the code. I don't feel good about removing it.
I also don't think that change belongs in a "Update coin control and smartfee labels" pull.

**Comment by MarcoFalke** [comment]
> @laanwj  I also don't think that change belongs in a "Update coin control and smartfee labels" pull.

Still, the fee label behaves wrong and needs update, but I force dropped the last commit to open a new PR for this...

**Pull request merged**